docs/introduction/Car-registry-tutorial.rst

```diff
@@ -309,29 +305,28 @@ AbstractRegularTransaction
 
 Basic functionality is implemented for building required unlockers for input Regular boxes and returning a list of output Regular boxes according to input parameter *outputRegularBoxesData*. Also, basic transaction semantic validity is checked here. 
 
-
 CarDeclarationTransaction 
 *************************
 
 *CarDeclarationTransaction* extends previously declared *AbstractRegularTransaction* in the following way: ``public final class CarDeclarationTransaction extends AbstractRegularTransaction``
-newBoxes() -- a new box with a newly created car shall be added as well. Thus that function shall be overridden as well for adding new CarBox additional to regular boxes.  
+newBoxes() -- a new box with a newly-created car shall be added as well. Thus, that function shall be overridden as well for adding new CarBox additional to regular boxes.  
 
 SellCarTransaction 
 ******************
 
-*SellCarTransaction* extends previously declared AbstractRegularTransaction in next way: ``public final class SellCarTransaction extends AbstractRegularTransaction``
-Similar to *CarDeclarationTransaction, newBoxes()* function shall also return a new specific box. In our case that new box is *CarSellOrderBox*. Also due we have specific box to open (CarBox), we also need to add unlocker for CarBox, so unlocker for that CarBox had been added in ``public List<BoxUnlocker<Proposition>> unlockers()``
+*SellCarTransaction* extends previously declared AbstractRegularTransaction in next (following?) way: ``public final class SellCarTransaction extends AbstractRegularTransaction``
+Similar to *CarDeclarationTransaction, newBoxes()* function shall also return a new specific box. In our case that new box is *CarSellOrderBox*. Also, since we have a specific box to open (CarBox), we also need to add unlocker for CarBox, so unlocker for that CarBox had been added in ``public List<BoxUnlocker<Proposition>> unlockers()``
```

Comment:
@zennermac 

Similar to the *CarDeclarationTransaction function, the newBoxes()* function will also return a new specific box. In our case that new box is a *CarSellOrderBox*. Since we have a specific box to open (CarBox), we also need to add an unlocker for CarBox. The unlocker for that CarBox had been added to the ``public List<BoxUnlocker<Proposition>> unlockers()``